ci/tests.py

```diff
@@ -204,6 +275,17 @@ def test_generate_report_registrations(self):
         # Receiver Identity
         self.add_identity_callback('receiver_id')
 
+        # Subscriptions, first page, just returns empty results to make sure
+        # we're actually paging through the results sets using the `next`
+        # parameter
+        self.add_blank_subscription_callback()
+
+        self.add_subscriptions_callback()
+
+        self.add_messageset_callback()
+
+        self.add_identity_callback('17cf37cf-edd6-4634-88e3-f793575f7e3a')
```

Comment:
For this test, can we not just add a blank callback, `self.add_blank_subscription_callback(next_=None)`, since we're not trying to test that sheet of the workbook here?